source/reference/minio-mc/mc-license-register.rst

```diff
@@ -61,15 +61,18 @@ Register a MinIO deployment at alias ``minio3`` on SUBNET that does not have dir
 
    mc license register minio3 --airgap
 
-#. Run the command to return a registration token
-#. Copy the registration token
-#. In a web browser, go to https://subnet.min.io and log in with your |SUBNET| credentials
-#. Select the  :guilabel:`Register` button
-#. Select :guilabel:`No` for the question :guilabel:`"Is the cluster connected to the internet?"`
-#. Paste the copied token into the box for :guilabel:`Register using MinIO Client Utility`
-#. Select :guilabel:`Register`
-#. Copy the API token that displays
-#. Back in the terminal, paste or enter the API token to complete the registration process
+#. Run the command to return a registration link with token
+#. Go to the copied registration link in a web browser and sign in to SUBNET, if necessary
```

Comment:
```suggestion
#. Open the copied registration link in a web browser and sign in to SUBNET
```

Reply:
I understand why the 'if necessary' is there, but I _think_ we are fine without it. If they get straight to SUBNET or not it works.

Reply:
Dispensing with unnecessary words for the win!